src/services/link/index.js

```diff
@@ -16,25 +16,49 @@ module.exports = function setup() {
 		lean: true,
 	};
 
+	const registrationLinkExpirationDays = app.get('registrationLinkExpirationDays') || 30;
```

Comment:
use config/globals.js to set default and import the final value from globals instead